src/polyswarm_api/core.py

```diff
@@ -328,6 +329,8 @@ def _delete_endpoint(cls, api, **kwargs):
     @classmethod
     def _params(cls, method, *param_keys, **kwargs):
         params = {}
+        if cls.QUERY_PARAM_SCOPE:
+            params = {scope: kwargs.get(scope) for scope in cls.QUERY_PARAM_SCOPE}
```

Comment:
I am not understanding where this `param` variable get's used, am I missing something?